styles.css

```diff
@@ -95,6 +95,68 @@ nav {
     margin: auto;
 }
 
+
+
+.wordInput {
+    height: 70px;
+    width: 70%;
+    margin: 0 auto;
+    margin-bottom: 25px;
+}
+
+form {
+    height: 100%;
+    display: flex;
+}
+.textInput {
+    font-family: var(--letterFont);
+    font-size: 3rem;
+    text-align: center;
+    width: 70%;
+    border-radius: 5px 0px 0px 5px;
+    border-style: solid;
+    border-color: #C6C6C6;
+    background-color: #ffffff;
+    margin-right: 0;
+}
+/* this styles the placeholder text */
+.textInput::placeholder {
+    color: #000000;
+    text-align: center;
+}
+
+.submitFormButton {
+    background-color: #47b863;
+    font-size: 3rem;
+    font-family: var(--letterFont);
+    width: 30%;
+    border-radius: 0px 5px 5px 0px;
+    border-color: #47b863;
+    border-style: solid;
+}
+.retryButton {
+    background-color: #f4eb47;
+    font-size: 3rem;
+    font-family: var(--letterFont);
+    width: 30%;
+    border-radius: 0px 5px 5px 0px;
+    border-color: #f4eb47;
+    border-style: solid;
+    display: none;
+}
+/* toggling this class completely removes the section*/
+.outcome {
+    /*display: none;*/
```

Comment:
Would it be better to leave this commented in for now and only add (or toggle) the class when it's needed?

Reply:
If I leave it in then the whole thing will disappear so I left it to aid the functionality